File: Programming Language(ver.2)/Go/604.go
package main
import (
	"fmt"
	"sort"
)

func main(){
	var array = make([]int, 9)
	for i := 0; i < 9; i++ {
		fmt.Scan(&array[i])
	}
	sort.Ints(array)
	var total = 0
	for i := 0; i < 9; i++ {
		fmt.Println(array[i])
		total += array[i]
	}
	fmt.Printf("sum = %d\n", total)
}

File: Programming Language(ver.2)/C++/604.cpp
#include <iostream>
using namespace std;

void sort(int[], int);
int main()
{
    int i, input[9], total = 0;
    for (i = 0; i < 9; i++) {
        cin >> input[i];
    }

    sort(input, 9);
    for (i = 0; i < 9; i++) {
        total += input[i];
        cout << input[i] << endl;
    }
    cout << "sum = " << total;
    return (0);
}

void sort(int list[], int length) {
    int i, j, n;
    for (i = 1; i < length; i++) {
        for (j = i; j > 0; j--) {
            if (list[j] < list[j - 1]) {
                n = list[j];
                list[j] = list[j - 1];
                list[j - 1] = n;
            }
        }
    }
}